### PR TITLE
Stop storing instead of duplicating analysis

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR makes cg throw an error if try to store an analysis that has already been created in status-db

How to test duplication check:
0. install on stage (add instructions on how to as needed)
1. us
1. find/create case that has been stored already
1. remove bundle in `housekeeper delete bundle [casename]`
1. `cg store analysis [path_to_analysis_config]`

Expected outcome:
cg should raise an AnalysisDuplicationError exception 

How to test normal behaviour:
0. install on stage (add instructions on how to as needed)
1. us
1. find/create case that has NOT been stored already
1. `cg store analysis [path_to_analysis_config]`

Expected outcome:
cg should store the analysis, i.e. create analysis record in status db and upload to housekeeper

### CODE REVIEW

File: cg/cli/store.py [hunk 1]
```diff
@@ -77,7 +77,10 @@ def _include_files_in_housekeeper(bundle_obj, context, hk_api, version_obj):
 def _add_new_complete_analysis_record(bundle_data, family_obj, status, version_obj):
 
     pipeline = family_obj.links[0].sample.data_analysis
-    pipeline = pipeline if pipeline else 'mip' # TODO remove this default from here
+
+    if status.analysis(family=family_obj, started_at=version_obj.created_at):
```
Comment (ingkebil) [inline cg/cli/store.py]: haha, not assigning it was the solution to the faulty query? :D

File: cg/exc.py [hunk 2]
```diff
@@ -14,6 +14,10 @@ class AnalysisNotFinishedError(CgError):
     pass
 
 
+class AnalysisDuplicationError(CgError):
+    """Error related to trying to create duplicate analysis objects"""
+
```
Comment (ingkebil) [inline cg/exc.py]:
```suggestion
    pass
```

File: cg/cli/store.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/cli/store.py]: Well, the faulty query was due to uninitiated mock object

**Comment by barrystokman** [comment]
Tested and approved!

File: cg/exc.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply cg/exc.py]: removed because of lint warning

**Pull request merged**